cl/lib/search_utils.py

```diff
@@ -1104,13 +1104,18 @@ async def get_related_clusters_with_cache(
     si = ExtraSolrInterface(settings.SOLR_OPINION_URL, mode="r")
 
     # Use cache if enabled
+    cache = caches["db_cache"]
```

Comment:
Can we also use this variable a few lines down instead of doing `caches["db_cache"]` again?

Reply:
Sure, I've applied the change. Setting auto-merge.